File: Day 02/p4.py
# 🔹 4. Function-Based Calculator
# Rebuild your calculator using a function named calculate(a, b, operator) that takes 3 inputs and returns the result.
# Let the user input two numbers and an operator and call the function to get the answer.





def calculate(a, b, operator):
    if operator == "+":
        return a + b
    elif operator == "-":
        return a - b
    elif operator == "*":
        return a * b
    elif operator == "/":
        return a / b
    else:
        return "Invalid"

a = float(input("Enter first number: "))
b = float(input("Enter second number: "))
operator = input("Enter operator: ")

print("Result:", calculate(a, b, operator))

File: Day 38/P4.js
// 4. Simple Calculator
// Write a function calculate(a, b, operator) that performs +, –, *, / based on the operator.



function calculate(a, b, operator) {
  if (operator === "+") {
    console.log(a + b);
  } else if (operator === "-") {
    console.log(a - b);
  } else if (operator === "*") {
    console.log(a * b);
  } else if (operator === "/") {
    console.log(a / b);
  } else {
    console.log("Invalid operator");
  }
}

calculate(1, 9, "+"); 
calculate(2,8,"/");
calculate(2,8,"-");
calculate(2,8,"*");
// node "Day 38/p4.js"